pkg/apis/pipeline/v1beta1/taskrun_validation.go

```diff
@@ -138,6 +140,19 @@ func validateStepOverrides(overrides []TaskRunStepOverride) (errs *apis.FieldErr
 	return errs
 }
 
+// validateTaskRunComputeResources ensures that compute resources are not configured at both the step level and the task level
+func validateTaskRunComputeResources(computeResources *corev1.ResourceRequirements, overrides []TaskRunStepOverride) (errs *apis.FieldError) {
+	for _, override := range overrides {
+		if override.Resources.Size() > 0 && computeResources != nil {
+			return apis.ErrMultipleOneOf(
+				"stepOverrides.resources",
+				"taskRunSpecs.computeResources",
```

Comment:
the field name here should just be `computeResources`

Reply:
so here mean we leave out the "parent" field, `taskRunSpecs`, for it? 
thinking it perhaps could help users locate the "error" field

Reply:
right! let me get rid of it. thanks Lee.